charts/tempo-distributed/values.yaml

```
@@ -32,13 +34,16 @@ tempo:
   # -- Common annotations for all pods
   podAnnotations: {}
   # -- SecurityContext holds pod-level security attributes and common container settings
-  securityContext: {}
-  #  capabilities:
-  #    drop:
-  #    - ALL
-  #  readOnlyRootFilesystem: true
-  #  runAsNonRoot: true
-  #  runAsUser: 1000
+  securityContext:
+    capabilities:
+      drop:
+        - ALL
+      readOnlyRootFilesystem: true
+      runAsNonRoot: true
+      runAsUser: 1000
+      runAsGroup: 1000
+      fsGroup: 1000
```

Need to remove extra indent:

```suggestion
    readOnlyRootFilesystem: true
    runAsNonRoot: true
    runAsUser: 1000
    runAsGroup: 1000
    fsGroup: 1000
```

See this CI Error below:
```
Error: INSTALLATION FAILED: unable to build kubernetes objects from release manifest: error validating "": error validating data: [ValidationError(Deployment.spec.template.spec.containers[0].securityContext.capabilities): unknown field "fsGroup" in io.k8s.api.core.v1.Capabilities, ValidationError(Deployment.spec.template.spec.containers[0].securityContext.capabilities): unknown field "readOnlyRootFilesystem" in io.k8s.api.core.v1.Capabilities, ValidationError(Deployment.spec.template.spec.containers[0].securityContext.capabilities): unknown field "runAsGroup" in io.k8s.api.core.v1.Capabilities, ValidationError(Deployment.spec.template.spec.containers[0].securityContext.capabilities): unknown field "runAsNonRoot" in io.k8s.api.core.v1.Capabilities, ValidationError(Deployment.spec.template.spec.containers[0].securityContext.capabilities): unknown field "runAsUser" in io.k8s.api.core.v1.Capabilities]
```